velero-plugin-for-gcp/volume_snapshotter.go

```diff
@@ -275,13 +272,14 @@ func (b *VolumeSnapshotter) createSnapshot(snapshotName, volumeID, volumeAZ stri
 	gceSnap := compute.Snapshot{
 		Name:        snapshotName,
 		Description: getSnapshotTags(tags, disk.Description, b.log),
+		SourceDisk:  disk.SelfLink,
 	}
 
 	if b.snapshotLocation != "" {
 		gceSnap.StorageLocations = []string{b.snapshotLocation}
 	}
 
-	_, err = b.gce.Disks.CreateSnapshot(b.snapshotProject, volumeAZ, volumeID, &gceSnap).Do()
+	_, err = b.gce.Snapshots.Insert(b.snapshotProject, &gceSnap).Do()
```

Comment:
Looks like the GCP disk snapshots are available globally. Although the old API needs zone as the parameter, the created snapshots are still global.
Without the API change, does it work with cross-project scenarios?

Reply:
Just got the multiple project GCP environment, and verified this PR. It worked.
I'm OK with this change. Although the previous GCP API might also work, this one looks more concise.

Another thing is that there was another cross-project scenario. The Velero is located in a separate project, which is different from the backup project and the restore project. I think this change makes the documented cross-project scenario work but will break the just-mentioned use case.

I think we can merge this PR first, then I will try to figure out how to make both scenarios work.